src/Speech/V1beta1/SpeechClient.php

```diff
@@ -0,0 +1,369 @@
+     * has been sent and processed.
+     *
+     * Sample code:
+     * ```
+     * try {
+     *     $speechClient = new SpeechClient();
+     *     $config = new RecognitionConfig();
+     *     $audio = new RecognitionAudio();
+     *     $response = $speechClient->syncRecognize($config, $audio);
+     * } finally {
+     *     if (isset($speechClient)) {
+     *         $speechClient->close();
+     *     }
+     * }
+     * ```
+     *
+     * @param RecognitionConfig $config       [Required] The `config` message provides information to the recognizer
+     *                                        that specifies how to process the request.
+     * @param RecognitionAudio  $audio        [Required] The audio data to be recognized.
+     * @param array             $optionalArgs {
+     *                                        Optional.
+     *
+     *     @type \Google\GAX\RetrySettings $retrySettings
+     *          Retry settings to use for this call. If present, then
+     *          $timeoutMillis is ignored.
+     *     @type int $timeoutMillis
+     *          Timeout to use for this call. Only used if $retrySettings
+     *          is not set.
+     * }
+     *
+     * @return \google\cloud\speech\v1beta1\SyncRecognizeResponse
+     *
+     * @throws \Google\GAX\ApiException if the remote call fails
+     */
+    public function syncRecognize($config, $audio, $optionalArgs = [])
+    {
+        $request = new SyncRecognizeRequest();
+        $request->setConfig($config);
+        $request->setAudio($audio);
+
+        $mergedSettings = $this->defaultCallSettings['syncRecognize']->merge(
+            new CallSettings($optionalArgs)
+        );
+        $callable = ApiCallable::createApiCall(
+            $this->speechStub,
+            'SyncRecognize',
+            $mergedSettings,
+            $this->descriptors['syncRecognize']
+        );
+
+        return $callable(
+            $request,
+            [],
+            ['call_credentials_callback' => $this->createCredentialsCallback()]);
+    }
+
+    /**
+     * Perform asynchronous speech-recognition: receive results via the
+     * google.longrunning.Operations interface. Returns either an
+     * `Operation.error` or an `Operation.response` which contains
+     * an `AsyncRecognizeResponse` message.
+     *
+     * Sample code:
+     * ```
+     * try {
+     *     $speechClient = new SpeechClient();
+     *     $config = new RecognitionConfig();
+     *     $audio = new RecognitionAudio();
+     *     $operationResponse = $speechClient->asyncRecognize($config, $audio);
+     *     $operationResponse->pollUntilComplete();
+     *     if ($operationResponse->operationSucceeded()) {
+     *       $result = $operationResponse->getResult();
+     *       // doSomethingWith($result)
+     *     } else {
+     *       $error = $operationResponse->getError();
+     *       // handleError($error)
+     *     }
+     * } finally {
+     *     if (isset($speechClient)) {
```

Comment:
nit: `$speechClient` will always be set, since `__construct` does not throw an exception.